src/data/animations/battle-anims.ts

```diff
@@ -129,19 +159,33 @@ export abstract class BattleAnim {
             frame.target === AnimFrameTarget.GRAPHIC
             && isReversed(this.srcLine[0], this.srcLine[2], this.dstLine[0], this.dstLine[2])
           ) {
-            scaleX = scaleX * -1;
+            zoomX = zoomX * -1;
           }
           break;
       }
       const angle = -frame.angle;
       const key = frame.target === AnimFrameTarget.GRAPHIC ? g++ : frame.target === AnimFrameTarget.USER ? u++ : t++;
-      ret.get(frame.target)!.set(key, { x: x, y: y, scaleX: scaleX, scaleY: scaleY, angle: angle }); // TODO: is the bang correct?
+      ret.get(frame.target)!.set(key, { x: x, y: y, zoomX: zoomX, zoomY: zoomY, angle: angle }); // TODO: is the bang correct?
```

Comment:
NIT: redundant. Probably due to refactor heck-meck

```suggestion
      ret.get(frame.target)!.set(key, { x, y, zoomX, zoomY, angle }); // TODO: is the bang correct?
```